### PR TITLE
fix: setIsAuth

### PR DESCRIPTION
**Pull request by LeAlencar**

### CODE REVIEW

**Comment by LeAlencar** [comment]
Fix some typos

File: src/context/AuthContext.tsx [hunk 1]
```diff
@@ -11,11 +11,10 @@ export const AuthContext = createContext({} as IAuthContext)
 
 export const AuthProvider = ({ children }: { children: ReactNode }) => {
   const token = GetToken()
-
   const [isAuth, setIsAuth] = useState(token ? true : false)
 
   useEffect(() => {
-    token ? setIsAuth(true) : localStorage.removeItem('login')
+    token ? setIsAuth(true) : setIsAuth(false)
```
Comment (fersilva16) [inline src/context/AuthContext.tsx:17]: remove token and logout user?

Comment (LeAlencar) [reply src/context/AuthContext.tsx:17]: Why should i try to remove the token if GetToken tries to get it from localStorage? I should only remove it when implementing logout, no?

Comment (fersilva16) [reply src/context/AuthContext.tsx:17]: Yeah, seems good

**Pull request merged**